### PR TITLE
Added redux structure

### PR DESCRIPTION
**Pull request by marcoalfonso**
Not finished yet, just opening to show the progress.

### CODE REVIEW

File: app/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+export function memberInfoFetchDataSuccess(memberInfo) {
+    return {
+        type: 'MEMBER_INFO_FETCH_DATA_SUCCESS',
+        memberInfo
+    };
+}
+
+
+export function memberInfoFetchData(userSessionCookie) {
```
Comment (matt-sm) [inline app/actions/index.js]: don't need this parameter anymore - browser will manage cookie for client

File: app/components/Header/Header.js [hunk 2]
```diff
@@ -1,29 +1,75 @@
...
+								<li>
+									{this.props.memberInfo.isAuthenticated ?
+										<span>{ this.dashBoardLink() }</span>
+										:
+										<a href="/signup">Join the Marketplace</a>
+									}
+								</li>
+								<li>
+									{this.props.memberInfo.isAuthenticated ?
+										<a  href="/logout">Sign out</a>
```
Comment (matt-sm) [inline app/components/Header/Header.js]: shouldn't this call /api/logout?

Comment (marcoalfonso) [inline app/components/Header/Header.js]: I could add the action, but we don't actually need it with our awesome routing :P ?

File: app/App.js [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
 
-export default App
\ No newline at end of file
+render (
+    <Provider store={store}>
+        <div id="Application" className="uikit-grid uikit-body">
+		  <header role="banner">
+			  	<Banner />
+			  	<GovAuBanner />
+		  		<Header />
```
Comment (dalehurwitz) [inline app/App.js]: Formatting please

Comment (marcoalfonso) [inline app/App.js]: Hahaha the code is open source, you can submit a pr 👅  💯 🔥

**Pull request merged**